scripts/generate-safe/script.js

```diff
@@ -4,12 +4,12 @@
 const fixture = [];
 
 function test(testcase, mode) {
-    var el = document.createElement('div');
```

Comment:
This module should be written in ES5 to able run it in old browsers without a transpilation. Please, revert changes. You might add a warning comment to avoid the same changes in future (or I will add such a comment by myself).

Reply:
Hmmm, the first line (besides) imports has a `const`, though. I have no idea how the whole thing works. Should I change that to `var` as well or only revert this file?

Reply:
Yes, change for var please. I suppose const works in IE11 somehow and I missed that.